app/views/admin/promotional_features/show.html.erb

```diff
@@ -1,16 +1,30 @@
-<% page_title "Promotional feature: #{@promotional_feature.title}" %>
-
-<%= content_tag_for(:div, @promotional_feature) do %>
-  <h1>Promotional feature: <%= @promotional_feature.title %></h1>
-  <p><%= link_to "Back to #{@organisation.name}", [:admin, @organisation, PromotionalFeature] %></p>
-
-  <%= render @promotional_feature.items %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: [:admin, @organisation, PromotionalFeature]
+  } %>
+<% end %>
+<% content_for :page_title, "Promotional feature: #{@promotional_feature.title}" %>
+<% content_for :context, "Promotional feature" %>
+<% content_for :title, @promotional_feature.title %>
+<% content_for :title_margin_bottom, 6 %>
 
-  <% if @promotional_feature.has_reached_item_limit? %>
-    <p><em>(maximum number of feature items reached)</em></p>
-  <% else %>
-    <div class="form-actions">
-      <%= link_to "Add feature item", new_admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature), class: "btn btn-lg btn-primary"%>
-    </div>
-  <% end %>
+<% if @promotional_feature.has_reached_item_limit? %>
+  <%= render "govuk_publishing_components/components/inset_text", {
+    text: "maximum number of feature items reached"
```

Comment:
I suspect this text should become "Title case" - it might be worth double-checking with @nnagewad 

```suggestion
    text: "Maximum number of feature items reached"
```